<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>metadataCreator</class>
 <widget class="QDialog" name="metadataCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Path To Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineOutPath"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnOutPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sign With User Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineUserID"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files To Import</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>save00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave00"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave00">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>save01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave01"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave01">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>save02</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave02"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave02">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>save03</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave03"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave03">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>save04</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave04"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave04">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>save05</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave05"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave05">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>save06</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave06"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave06">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>save07</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave07"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave07">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>save08</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave08"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave08">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>save09</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSave09"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSave09">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icon/quit</normaloff>:/icon/quit</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
